tensorboard/webapp/plugins/npmi/views/data_selection/metric_search/metric_search_module.ts

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+
+import {MatIconModule} from '@angular/material/icon';
+import {MatAutocompleteModule} from '@angular/material/autocomplete';
+import {MatInputModule} from '@angular/material/input';
+
+import {MetricSearchComponent} from './metric_search_component';
+import {MetricSearchContainer} from './metric_search_container';
+
+@NgModule({
+  declarations: [MetricSearchComponent, MetricSearchContainer],
+  imports: [
+    CommonModule,
+    FormsModule,
```

Comment:
Similarly here, what exports from FormsModule are we depending on, in MetricSearchComponent/Container?

Reply:
Removed.